src/slurm_plugin/clustermgtd.py

```diff
@@ -950,6 +960,21 @@ def _find_active_nodes(partitions_name_map):
                 active_nodes += partition.slurm_nodes
         return active_nodes
 
+    @staticmethod
+    def _get_unhealthy_ice_nodes(unhealthy_dynamic_nodes: list) -> dict:
```

Comment:
optional: when setting the typing info to a collection (e.g. list or dict) you could also add typing info on the items of that collection

Reply:
Will do tomorrow

Reply:
Done